services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/multipart/MultipartS3AsyncClient.java

```diff
@@ -62,11 +64,13 @@ private MultipartS3AsyncClient(S3AsyncClient delegate, MultipartConfiguration mu
 
     @Override
     public CompletableFuture<PutObjectResponse> putObject(PutObjectRequest putObjectRequest, AsyncRequestBody requestBody) {
```

Comment:
While we are at it, can we enable it for GetObjectRequest by default?

Reply:
multipart download not supported yet, adding TODO